packages/styles/scss/components/data-table/_data-table-sort.scss

```
@@ -0,0 +1,240 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@use '../../config' as *;
+@use '../../motion' as *;
+@use '../../spacing' as *;
+@use '../../theme' as *;
+@use '../../utilities/button-reset';
+@use '../../utilities/convert' as *;
+@use '../../utilities/focus-outline' as *;
+
+/// Data table sort styles
+/// @access private
```

For mixins that aren't hypen-prefixed we'll have to treat them as public unless we're saying we don't want teams to include this file directly 👀 

```suggestion
/// @access public
```

---

Fixed 👍